src/www/csrf.inc

```diff
@@ -96,13 +96,11 @@ class LegacyCSRF
             // csrf token for Ajax type requests
             $script = "
             <script>
-              $( document ).ready(function() {
```

Comment:
is this a requirement for the `customElements.define(....)` ? at a first glance this changes loading behaviour, but maybe I'm mistaken here. If it's not related, we should probably keep it as is now.

Reply:
I get a race condition. The element will initialize when it is ready by itself. At that point the full document is not yet loaded. This way we do not waste time waiting for unnecessary elements being loaded.

Since the boot grid search will be executed before the document is loaded, the post call is missing the csrf token andthe token BTW. does not depend on the DOM being constructed. So it should not be delayed.